browser/components/CLAEditor.tsx

```diff
@@ -62,10 +62,21 @@ export default class CLAEditor extends React.Component<Props, State> {
 
   componentWillReceiveProps(nextProps) {
     this.setState({
-      data: nextProps.data as State['data'],
+      data: {
+        ...(nextProps.data as State['data']),
+        signed_date: this.handleDate(this.props.data.signed_date),
+        approved_date: this.handleDate(this.props.data.approved_date),
+      },
     });
   }
 
+  handleDate(d) {
```

Comment:
Could you add a comment here (in the code) explaining what this function is doing? I'm assuming this is slicing off the timestamp part, but can't quite tell.